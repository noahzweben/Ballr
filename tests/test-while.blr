gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

		int i = 5;
		while (i > 0) {
 	      print(i);
  	    i = i - 1;
		}	
  	print(42);	

  }
}