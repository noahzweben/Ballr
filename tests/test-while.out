5
4
3
2
1
42
SDL could not initialize! SDL Error: No available video device
Window initialization failed.